README.md

```diff
@@ -172,25 +176,16 @@ const { createFocusTrap } = require('../../dist/focus-trap');
 const container = document.getElementById('default');
 
 const focusTrap = createFocusTrap('#default', {
-  onActivate: function () {
-    container.className = 'trap is-active';
-  },
-  onDeactivate: function () {
-    container.className = 'trap';
-  },
+  onActivate: () => container.classList.add('is-active'),
+  onDeactivate: () => container.classList.remove('is-active'),
```

Comment:
Using `classList.add` and `classList.remove` are less error prone methods of adding/removing the "is-active" class.

We should use `classList` to endorse good coding practices.